src/overlays/actors/ovl_En_Yb/z_en_yb.h

```diff
@@ -7,12 +7,24 @@ struct EnYb;
 
 typedef void (*EnYbActionFunc)(struct EnYb*, GlobalContext*);
 
+#define ENYB_LIMBCOUNT 0x16
+
 typedef struct EnYb {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x2DC];
-    /* 0x0420 */ EnYbActionFunc actionFunc;
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s limbDrawTbl[ENYB_LIMBCOUNT];
+    /* 0x??? */ char weirdPadding[0x94]; // this is too much padding what
```

Comment:
```suggestion
    /* 0x??? */ UNK_TYPE1 weirdPadding[0x94]; // this is too much padding what
```
I would use the UNK_TYPE1 instead of char, and not use `weirdPadding` as the name.

Reply:
Wonder if this has something to do with Player's limb tables.

Reply:
I assume it has to do with the skeleton needng to be player compatible, yeah, although I assumed we didn't have the types to actually fill it out correctly yet